mne/io/base.py

```diff
@@ -2570,6 +2570,15 @@ def concatenate_raws(raws, preload=None, events_list=None, *,
         return raws[0], events
 
 
+def match_channel_orders(raws, copy=True):
+    """Ensure consistent channel order across raws."""
+    raws = deepcopy(raws) if copy else raws
+    ch_order = raws[0].ch_names
+    for raw in raws[1:]:
+        raw.reorder_channels(ch_order)
+    return raws
+
+
```

Comment:
Is this fine?

Reply:
needs a proper docstring (with `Parameters` and `Returns` sections at least). Probably should also add it here:

https://github.com/mne-tools/mne-python/blob/57f5ce300f8e67ffcc0d9a19ab724693105b74b6/mne/__init__.py#L38

and here:

https://github.com/mne-tools/mne-python/blob/57f5ce300f8e67ffcc0d9a19ab724693105b74b6/doc/sensor_space.rst?plain=1#L11